thavalon-server/src/game/builder.rs

```diff
@@ -3,10 +3,10 @@
 use tokio::sync::mpsc;
 use tokio::task;
 
-use super::Game;
 use super::engine;
 use super::interactions::ChannelInteractions;
 use super::messages::{Action, Message};
+use super::Game;
```

Comment:
Question: why did `super::Game` move down? Is this a Rust style thing I should be following?

Reply:
I’m not sure, that might’ve been VSCode or rustfmt. I can undo it

Reply:
Checked and it looks like it's rustfmt. I thought it didn't reorder imports, but they might've changed that